packages/expo/cli/utils/user/actions.ts

```diff
@@ -0,0 +1,99 @@
+import assert from 'assert';
+import chalk from 'chalk';
+
+import * as Log from '../../log';
+import { ApiV2Error } from '../api';
+import { CommandError } from '../errors';
+import { learnMore } from '../link';
+import promptAsync, { Question } from '../prompts';
+import { retryUsernamePasswordAuthWithOTPAsync } from './otp';
+import { Actor, getUserAsync, loginAsync } from './user';
+
+/** Show login prompt while prompting for missing credentials. */
```

Comment:
Anyone: _writes comment_
James:
```suggestion
/** Show log in prompt while prompting for missing credentials. */
```

Reply:
Sorry, couldn't let this one slip 😂

Reply:
It's actually 'login prompt'. The verb is 'to log in' (separate words), but the noun is 'login'.

Reply:
Kim is right. "The setup steps", "Set up your project"

Reply:
jeez

![its-a-meme](https://user-images.githubusercontent.com/1203991/151408258-b046e135-5a5d-4df1-9c30-8dd1b39ce7b5.gif)